docs/source/querying_data/dedup.md

```diff
@@ -62,5 +64,3 @@ In order for the `dedup-response` processor to do its job, it must be able to ac
   }
 }
 ```
-
-The practical effect of this is that it's possible to enable or disable NDD by controlling the contents of `_source`.
```

Comment:
I think this is useful information.  I certainly make use of it, as does the NDD example notebook.